### ISSUE TITLE
Troubleshooting easy access site needs photo guide

### ISSUE THREAD

**Issue by CobyKabili** [Robo3D/support.robo3d.com#29]
Better if we had photos that showed the problem alongside the description

For Inspiration: http://support.3dverkstan.se/article/23-a-visual-ultimaker-troubleshooting-guide

**Comment by mnacinopa** [comment]
I think we should start tackling this and make use of the "General" Section on the help site.

**Comment by CobyKabili** [comment]
agreed. We also need to put in this section how to find your order status and updated your address and things of this nature
